README.md

```diff
@@ -100,3 +100,9 @@ Accessing OKD UI
 ```
 TODO - in the process of working out the details here.
 ```
+
+In order to check newly created provider run,
+this will point to the local created provider upon cluster-up
```

Comment:
Sorry, in afraid I still don't understand. What "created provider run" and "local created provider"? What does the check do? When would I use it?

Reply:
after some create a new provider, using ./cluster-provision/ocp/4.3/provision.sh
for example, it creates a container, runs openshift-installer in it, shutdown machines, and then commit the image (locally), before publishing it to quay, we should check cluster-up with that local image instead of taking the official one from quay.

another case is for example if we want to alter provision.sh to fix bugs / extend it with another rhel 7.7 machine, we should check it

this export allow us to quick point on the local provisioned container
it existed for k8s (with little problems that are addressed as well), i added it in a single interface for okd/ocp as well